x-pack/filebeat/module/fortinet/clientendpoint/test/generated.log-expected.json

```diff
@@ -0,0 +1,2302 @@
+        "observer.vendor": "Fortinet",
+        "rsa.internal.messageid": "ms-wbt-server",
+        "rsa.time.event_time": "2017-04-30T02:00:00.000Z",
+        "service.type": "fortinet",
+        "tags": [
+            "fortinet.clientendpoint",
+            "forwarded"
+        ]
+    },
+    {
+        "@timestamp": "2017-05-14T02:00:00.000Z",
+        "event.code": "http",
+        "event.dataset": "fortinet.clientendpoint",
+        "event.module": "fortinet",
+        "event.original": "May 14 2017/05/14 tanimid3337.mail.corp proto=ipv6-icmp service=http status=deny src=10.217.150.196 dst=10.105.91.31 src_port=2056 dst_port=5987 server_app=loreme pid=853 app_name=psumquia traff_direct=ven block_count=660 logon_user=siutali@amnih2718.internal.example msg=failure",
+        "fileset.name": "clientendpoint",
+        "input.type": "log",
+        "log.flags": [
+            "dissect_parsing_error"
+        ],
+        "log.offset": 9002,
+        "observer.product": "FortiClient",
+        "observer.type": "Anti-Virus",
+        "observer.vendor": "Fortinet",
+        "rsa.internal.messageid": "http",
+        "rsa.time.event_time": "2017-05-14T02:00:00.000Z",
+        "service.type": "fortinet",
+        "tags": [
+            "fortinet.clientendpoint",
+            "forwarded"
+        ]
+    },
+    {
+        "@timestamp": "2017-05-29T02:00:00.000Z",
+        "event.code": "https",
+        "event.dataset": "fortinet.clientendpoint",
+        "event.module": "fortinet",
+        "event.original": "May 29 2017/05/29 laudant6813.mail.home proto=icmp service=https status=deny src=10.184.18.202 dst=10.226.83.168 src_port=5780 dst_port=4153 server_app=molli pid=4306 app_name=aturauto traff_direct=eturadi block_count=2512 logon_user=borios@rsitvolu3751.mail.lan msg=success",
+        "fileset.name": "clientendpoint",
+        "input.type": "log",
+        "log.flags": [
+            "dissect_parsing_error"
+        ],
+        "log.offset": 9282,
+        "observer.product": "FortiClient",
+        "observer.type": "Anti-Virus",
+        "observer.vendor": "Fortinet",
+        "rsa.internal.messageid": "https",
+        "rsa.time.event_time": "2017-05-29T02:00:00.000Z",
+        "service.type": "fortinet",
+        "tags": [
+            "fortinet.clientendpoint",
+            "forwarded"
+        ]
+    },
+    {
+        "@timestamp": "2017-06-12T02:00:00.000Z",
+        "event.code": "https",
+        "event.dataset": "fortinet.clientendpoint",
+        "event.module": "fortinet",
+        "event.original": "June 12 2017/06/12 mquelau5326.mail.lan proto=icmp service=https status=deny src=10.255.39.252 dst=10.113.95.59 src_port=863 dst_port=4367 server_app=fugitsed pid=1693 app_name=idolo traff_direct=luptate block_count=2612 logon_user=atisun@esci7741.www.host msg=success",
+        "fileset.name": "clientendpoint",
+        "input.type": "log",
+        "log.flags": [
+            "dissect_parsing_error"
```

Comment:
looks like pretty much all of these have parsing errors?

Reply:
fixed